src/one_dragon/base/controller/pc_controller_base.py

```
@@ -269,6 +311,173 @@ def _set_cursor_to(self, hwnd: int, cx: int, cy: int) -> None:
+        while self._mouse_tracker_running:
+            self._mouse_tracker_paused.wait()
+            if not self._mouse_tracker_running:
+                break
+            try:
+                hwnd = self.game_win.get_hwnd()
+                if hwnd:
+                    if win32gui.GetForegroundWindow() != hwnd or win32gui.IsIconic(hwnd):
+                        self._apply_tracker_pseudo_minimize(hwnd)
+                    self._align_tracker_anchor_to_cursor(hwnd)
+            except Exception:
+                pass
```

Ruff (0.15.6)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)